### PR TITLE
Update avm2 progress

### PR DESCRIPTION
**Pull request by Dinnerbone**
Max points changed from 4564 to 4560 (**-4**)
Impl points changed from 3086 to 3125 (**+39**)
Stub penalty changed from 285 to 300 (**+15**)

Overall we're up to 62% (**+1%**) implemented, 7% (**+1%**) stubbed.

### CODE REVIEW

File: assets/report.json [hunk 1]
```diff
@@ -4546,10 +4525,14 @@
     "flash.display.FocusDirection": {
```
Comment (Lord-McSweeney) [inline assets/report.json:4525]: This class is AIR-only and should be removed from the list.

File: assets/report.json [hunk 2]
```diff
@@ -5336,12 +5314,10 @@
     "": {
       "summary": {
         "max_points": 16,
-        "impl_points": 15,
+        "impl_points": 16,
         "stub_penalty": 0
       },
-      "missing": [
-        "isXMLName()"
```
Comment (Lord-McSweeney) [inline assets/report.json:5343]: This is stubbed right now- from the last PR, it seems that your tool doesn't mark methods that aren't defined on classes (e.g. `flash.crypto.getRandomBytes`, `flash.net.registerClassAlias`) as stubs.

File: assets/report.json (hunk 1 above)
Comment (Dinnerbone) [reply assets/report.json:4525]: Removed from the avm2 spec

File: assets/report.json (hunk 2 above)
Comment (Dinnerbone) [reply assets/report.json:5343]: Fixed

**Pull request merged**